### PR TITLE
Fix translation relative to the hostname validator

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6038

### CODE REVIEW

**Review by DavidePrincipi — APPROVED** [review]
Let's ask @gsanchietti and @edospadoni

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -199,7 +199,7 @@
     "user_already_exists": "An account with the same name already exists",
     "user_not_found": "Account not found",
     "valid_username": "First character must be a letter, than only lower letters, numbers and symbols like '-' and '_'",
-    "valid_hostname_0_9223372036854775807": "Must be a valid hostname or a Fully qualified domain name"
+    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a resource path component beginning with '/'"
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]:
```suggestion
    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a path beginning with '/'"
```

File: ui/public/i18n/language.json [hunk 2]
```diff
@@ -177,7 +177,7 @@
     "more_info_about": "More info about",
     "virtualhost": "Virtual hosts",
     "Reverse_Proxy": "Reverse Proxy",
-    "path-and-virtual-host-rules": "If you want to create an URL path rule, the name field must start with a '/'. If you need to create a virtual host name rule, then the name field must match a virtual host name",
+    "path-and-virtual-host-rules": "If you want to create a resource path matching rule, then enter the resource path beginning with a '/'. If you want to create a web site name matching rule, then type the web site fully qualified domain name.",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json:180]: I prefer the old label

Comment (DavidePrincipi) [reply ui/public/i18n/language.json:180]: - The old string says "URL path" rule. The manual says "resource path". I want to use the same wording.
- The old string says `virtual host` and that can be misleading. It already happened that people go and create a virtual host.

File: ui/public/i18n/language.json (hunk 1 above)
Comment (DavidePrincipi) [reply ui/public/i18n/language.json]: It's not a "generic" path, it's a [resource path](https://en.wikipedia.org/wiki/URL#Syntax), just one _segment_. 

- `/A` is good
- `/A/B` is wrong

```suggestion
    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a resource path segment beginning with '/'"
```

**Comment by gsanchietti** [comment]
I still prefer simpler labels.
But since I didn't face the issue, I can't offer a better point of view.

Feel free to commit labels you prefer ;)

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**